src/coreclr/src/System.Private.CoreLib/src/System/Collections/Generic/ArraySortHelper.CoreCLR.cs

```diff
@@ -20,6 +20,7 @@ internal partial class ArraySortHelper<T>
 
         public static IArraySortHelper<T> Default => s_defaultArraySortHelper;
 
+        [PreserveDependency(".ctor", "System.Collections.Generic.GenericArraySortHelper`1", "System.Private.CoreLib")]
```

Comment:
(Minor) You shouldn’t need to specify the assembly name. If you leave it off, the linker assumes the current assembly.

Reply:
Good to know, will fix.